app-guides/mysql-on-fly.html.markerb

```diff
@@ -77,7 +75,8 @@ primary_region = "bos"
   cpus = 1
   memory_mb = 2048
 
-# Add the following sections
+# Add the following sections for 8+, 
+# Note that the [processes] section would be different for 8.4, and 5.7 ( the syntax will be shown below )
```

Comment:
What if we put all three in the toml file, with 2 of them commented out? Something like:
```suggestion
# The [processes] section is different for 8+, 8.4, and 5.7. Use the one that matches your version.
# Use the following for versions 8 to 8.3:
[processes]
  app = """--datadir /data/mysql \
    --default-authentication-plugin mysql_native_password"""

# Uncomment and use the following for 8.4:
# [processes]
#  app = """--datadir /data/mysql \
#    --mysql-native-password=ON"""

# Uncomment and use the following for 5.7:
# [processes]
#  app = "--datadir /data/mysql"

# Add the following sections for all versions
[mounts]
  source = "mysqldata"
  destination = "/data"

[env]
  MYSQL_DATABASE = "some_db"
  MYSQL_USER = "non_root_user"```